src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -735,19 +735,15 @@ export class ObjectSettingWidget extends AbstractListSettingWidget<IObjectDataIt
 	}
 
 	protected renderHeader() {
-		if (this.model.items.length > 0) {
```

Comment:
If we are starting a brand new table, there isn't enough space to render the new row properly. I think we should always render the header.

## Before

![image](https://user-images.githubusercontent.com/8235156/86064041-30ee1400-ba3a-11ea-9eac-2c04a0656193.png)

## After

![image](https://user-images.githubusercontent.com/8235156/86064087-45321100-ba3a-11ea-8682-af270203f060.png)